examples/get_sensor_history.py

```diff
@@ -69,15 +69,20 @@ def main():
                        args.start).strftime('%Y-%m-%dT%H:%M:%SZ'),
                    datetime.utcfromtimestamp(args.end).strftime('%Y-%m-%dT%H:%M:%SZ')))
         if len(sensor_names) == 1:
-            # Request history for just a single sensor - result is timestamp, value, status
-            #    If value timestamp is also required, then add the additional argument: include_value_ts=True
+            # Request history for just a single sensor - result is
+            # timestamp, value, status
+            #    If value timestamp is also required, then add the additional argument:
+            #        include_value_ts=True
             #    result is then timestmap, value_timestmap, value, status
+            additional_fields = args.additional_fields
```

Comment:
Please test this example.  Doesn't it break here?
Test the get_sensor_info.py example as well, and confirm that the output is identical to that shown in its comments.